### ISSUE TITLE
Extracting predicted text from locally trained model

### ISSUE THREAD

**Issue by mustafakucuk0** [reppertj/image-captioning#1]
Hi! Thank you for this precious repo. I want to use this model in the one of my projects but I did not understand how you extract predicted text from images by using a locally trained model. I trained my model and it saved the 10 best models as ckpt files but as far as I understand you did not give any particular example about how to use a locally trained model. Am I going to use ckpt file as a model or save Trainer as the model pkl? 

Thank you again for this repo.

**Comment by reppertj** [comment]
Hey, looks like you might have figured it out, but for an example, check out [this notebook](https://colab.research.google.com/github/reppertj/image-captioning/blob/master/notebooks/visualizing_attention.ipynb). This could definitely be a bit friendlier-to-use for inference.

You'll have to create a datamodule, as in the notebook, then you can load the weights from the saved checkpoint: `model = CaptioningRNN.load_from_checkpoint(checkpoint_path, datamodule=datamodule, batch_size=1)`

Make sure you don't forget to call `model.eval()` to turn off batch norm & dropout & run the same preprocessing steps as in the notebook.

Feel free to ask if you have any questions!
